pandas/core/generic.py

```diff
@@ -4688,6 +4688,23 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
         Returns
         -------
         wh : same type as caller
+
+        Notes
+        -----
+        Because it is a function attribute, the signature for `DataFrame.where`
+        differs from `numpy.where`. Roughly `df1.where(m, df2)` is equivalent
+        to `np.where(m, df1, df2`.
+
+        Examples
+        ________
+        >>> df = pd.DataFrame(np.arange(10).reshape(-1, 2), columns=['A', 'B'])
+        >>> df.where(df %% 3 == 0, -df)
+           A  B
+        0  0 -1
+        1 -2  3
+        2 -4 -5
+        3  6 -7
+        4 -8  9
         """)
```

Comment:
can you show a compat example with np.where (e.g. show with DataFrame.where and np.where)